src/seedu/addressbook/parser/Parser.java

```diff
@@ -97,12 +87,25 @@ public Command parseCommand(String userInput) {
         case ExitCommand.COMMAND_WORD:
             return new ExitCommand();
 
+        case ContainCommand.COMMAND_WORD:
+            return prepareContain(arguments);
+
         case HelpCommand.COMMAND_WORD: // Fallthrough
+
         default:
             return new HelpCommand();
         }
     }
 
+    private Command prepareContain(String args) {
```

Comment:
Would be nice if this function has explanatory comments as well, see the `prepareAdd` function below :)